draft-ietf-quic-load-balancers.md

```diff
@@ -1298,8 +1300,8 @@ routing config rotation codepoint.
 The Shared-State Retry Service defined in {{shared-state-retry}} describes the
 format of retry tokens or new tokens protected and encrypted using AES128-GCM.
 Each token includes a 96 bit randomly generated unique token number, and an 8
-bit identifier of the AES-GCM encryption key. There are three important security
-considerations for these tokens:
+bit identifier of the AES-GCM encryption context. There are three important
+security considerations for these tokens:
 
```

Comment:
Context is a better word, since it encompasses both Key and IV. But if we do that, it might be good to say something like "AES-GCM encryption context containing Key and IV".

Reply:
OK, I have fix it, just like this:
```
and an 8
bit identifier used to get the AES-GCM encryption context. The AES-GCM encryption
context contains a 96 bit key and a 96 bit AEAD IV
```

Reply:
> Context is a better word, since it encompasses both Key and IV. But if we do that, it might be good to say something like "AES-GCM encryption context containing Key and IV".

OK, I have fix it, just like this:
```
and an 8 bit identifier used to get the AES-GCM encryption context. The AES-GCM encryption
context contains a 96 bit key and a 96 bit AEAD IV
```

Reply:
The key is 128 bits for AES_GCM 128.

Reply:
sorry, I made a foolish mistake...fixed